### PR TITLE
Issue93

### LINKED ISSUE #93: Number of attempts for a specific question is broken

### PR DESCRIPTION
**Pull request by ahmedqarmout2**
fixes #93

### CODE REVIEW

**Review by esteeroramy — APPROVED** [review]

File: server/questions.js [hunk 1]
```diff
@@ -38,9 +38,12 @@ exports.addQuestionByType = function(qType, question, callback) {
...
 	questionToAdd.visible = (question.visible === 'true');
-	questionToAdd.attempted = [];
-	questionToAdd.answered = [];
-	questionToAdd.attempts = [];
+	questionToAdd.correctAttempts = [];
+	questionToAdd.wrongAttempts = [];
+	questionToAdd.totalAttempts = [];
+	questionToAdd.correctAttemptsCount = 0;
+	questionToAdd.wrongAttemptsCount = 0;
+	questionToAdd.totalAttemptsCount = 0;
```
Comment (mehtadarshan1) [inline server/questions.js:56]: this will need to be updated in common.js in the questionAttribute object

**Review by Mohammed-JS-Khan — APPROVED** [review]

File: server/questions.js (hunk 1 above)
Comment (ahmedqarmout2) [reply server/questions.js:56]: Good catch I will update that tonight

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Nice work

File: views/question.pug [hunk 2]
```diff
@@ -5,20 +5,13 @@ block styles
...
 block title
-  if !preview
-    title Quizzard - Question
-  else
-    title Quizzard - Question Preview
+  title Quizzard - Question
 
 block content
-  if !preview
-    include navbar
```
Comment (esteeroramy) [inline views/question.pug:20]: This should not be removed

**Review by esteeroramy — CHANGES_REQUESTED** [review]

File: views/question.pug (hunk 2 above)
Comment (ahmedqarmout2) [reply views/question.pug:20]: fixed

**Review by esteeroramy — APPROVED** [review]

**Pull request merged**